runtime/syntax/cpp.yaml

```diff
@@ -2,7 +2,7 @@ filetype: c++
 
 detect:
     filename: "(\\.c(c|pp|xx)$|\\.h(h|pp|xx)?$|\\.ii?$|\\.(def)$)"
-    signature: "namespace|template|public|protected|private"
+    signature: "namespace|class|public|protected|private|template|constexpr|noexcept|nullptr|try|throw"
```

Comment:
This should be in a separate commit.
Also "try" might be too common word.
Also I believe all this should be guarded with `\b`.